opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -2974,7 +2976,8 @@ export const elPaginate = async (context, user, indexName, options = {}) => {
     );
 };
 export const elList = async (context, user, indexName, opts = {}) => {
-  const { first = ES_DEFAULT_PAGINATION, maxSize = undefined } = opts;
+  const { maxSize = undefined } = opts;
+  const first = opts.first ?? ES_DEFAULT_PAGINATION;
```

Comment:
what is the difference between `const { first = ES_DEFAULT_PAGINATION } = opts;` and `const first = opts.first ?? ES_DEFAULT_PAGINATION;` ?

Reply:
The null handling. { first = ES... } does not apply if first is null.